test/servinge2e/kourier/servicemesh_test.go

```diff
@@ -636,7 +639,7 @@ func TestKsvcWithServiceMeshJWTDefaultPolicy(t *testing.T) {
 
 func lookupOpenShiftRouterIP(ctx *test.Context) net.IP {
 	// Deploy an auxiliary ksvc accessible via an OpenShift route, so that we have a route hostname that we can resolve
-	aux := test.Service("aux", test.Namespace, image, nil)
```

Comment:
You seem to be changing the image from "quay.io/openshift-knative-serving-test/helloworld:v1.3" to "quay.io/openshift-knative-serving-test/hello-openshift:latest" . Is this expected? It seems strange

Reply:
This image https://github.com/openshift-knative/serverless-operator/blob/dd914d4643bb0287ce6f5c452e1f83fa91361b37/test/servinge2e/kourier/servicemesh_test.go#L338 is not multi-arch, so we have replaced it with multi-arch image(quay.io/openshift-knative-serving-test/hello-openshift:latest).

Reply:
OK. I'm curious if the tests will pass.